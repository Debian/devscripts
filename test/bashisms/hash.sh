#!/bin/sh

: BASHISM; hash
hash which # BASHISM
hash -r # BASHISM
