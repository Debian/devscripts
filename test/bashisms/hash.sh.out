possible bashism in bashisms/hash.sh line 3 (hash):
: BASHISM; hash
possible bashism in bashisms/hash.sh line 4 (hash):
hash which # BASHISM
possible bashism in bashisms/hash.sh line 5 (hash):
hash -r # BASHISM
